includes/definitions/procera.yaml

```diff
@@ -11,3 +11,5 @@ processor_stacked: 1
 discovery:
    - sysObjectId:
        - .1.3.6.1.4.1.15397.2
+mib_dir:
```

Comment:
MIB dirs matching the OS name are added by default.

Reply:
Well that's convenient. I'll remove it, thanks Murrant.